seqr/views/utils/individual_utils.py

```diff
@@ -185,3 +186,24 @@ def get_parsed_feature(feature):
             feature_json[field] = feature[field]
 
     return feature_json
+
+
+def get_updated_response_json(updated_individuals, updated_families, updated_notes, user):
```

Comment:
this is a very unclear function name. Maybe do something more like `get_updated_pedigree_json`